extensions/dicom-seg/src/components/SegmentItem.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { TableListItem, Icon } from '@ohif/ui';
+
+const ColouredCircle = ({ color }) => {
+  return (
+    <div
+      className="segment-color"
+      style={{ backgroundColor: `rgba(${color.join(',')})` }}
+    ></div>
+  );
+};
+
+const SegmentItem = ({ index, label, onClick, itemClass, color }) => (
+  <TableListItem
+    key={index}
+    itemKey={index}
+    itemIndex={index}
+    itemClass={itemClass}
+    itemMeta={<ColouredCircle color={color} />}
+    itemMetaClass="segment-color-section"
+    onItemClick={onClick}
+  >
+    <div>
+      <div className="segment-label" style={{ marginBottom: 4 }}>
+        {label}
+      </div>
+      {false && <div className="segment-info">{'...'}</div>}
+      {false && (
```

Comment:
Shipping some UI dark for future edit mode?

Reply:
Yes, maybe we should just take it out?

Reply:
Nah. It's fine. Maybe a quick comment.